dashboard/backend/handler/apihandler.go

```diff
@@ -0,0 +1,170 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+
+	restful "github.com/emicklei/go-restful"
+	"github.com/tensorflow/k8s/dashboard/backend/client"
+	"github.com/tensorflow/k8s/pkg/spec"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/pkg/api/v1"
+)
+
+type APIHandler struct {
+	cManager client.ClientManager
+}
+
+//add tensorboard ips, add azure file / gs links
```

Comment:
space after "//"

Needs a doc string; see https://golang.org/doc/effective_go.html#commentary

The first sentence should begin with the name of the thing being described; e.g.

"TfJobDetail is ...."